src/contracts/ApesNFT-address.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
```

Comment:
I don't think that you should include this change. Do not include contract data unless there is a new deployment and into a separate PR. Be very careful with these changes since the app will break as soon as this PR gets merged

Reply:
Sorry, I didn't check it. Let me change it